src/pages/dashboard/monitor/components/Charts/autoHeight.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+export type IReactComponent<P = any> =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)